source/layout.html

```diff
@@ -138,9 +138,9 @@ <h1 class="custom-page-header page-title has-dash">{{ page.name }}</h1>
         {% endif %}
       </div>
     {% endif %}
-    <footer>
+    <footer role="contentinfo">

```

Comment:
I'm interested in hearing what you experience when you run through this locally. I heard the screen reader say, "contentinfo" using ChromeVox, but I still didn't hear VO say it. Instead, it jumps right into the footer navigation and announces the first link. Do you think that makes sense? (It worked as I expected it to when I tested out the Lunch Break PR-- I am not sure what is different).

However, VO is detecting the contentinfo as a landmark in its rotor (which is so so cool btw):
![IMG_20191108_101809](https://user-images.githubusercontent.com/13512356/68488198-85762200-0212-11ea-82f2-8f237ea61410.jpg)

Reply:
Also I could not figure out how to take a screenshot when VO was open, so I had to take a pic with my phone and slack it to myself 😆

Reply:
I'm pretty sure I got it to say contentinfo in Voice Over, but I'll double check! +1 from me with this PR though